_pages/en/strategy-build-vs-buy.md

```diff
@@ -118,12 +134,13 @@ Stakeholders are expected to still comply with existing policy instruments inclu
 ### Legal
 
 1. IT Solutions use the ESDC Software Legal Terms Guidance (to be created - see [Coherent set of actions](#coherent-set-of-actions))
-2. IT Solutions and IT Products comply with the ESDC OSS Management Framework
+2. IT Solutions and IT Products comply with the ESDC Open Source Software Management Framework when applicable.
 
 ### Architecture
 
-1. IT Solutions and IT products loose coupling boundaries are at the Business Capability Model level.
-   - Duplication of IT Products or IT Solutions is supported as long as they are for different business capabilities.
+1. IT Solutions and IT products loose coupling boundaries are at intersection of value stream, value stage and business capability (ESDC Business Capability Model).
+   - Duplication of IT Solutions or IT Products is supported as long as they are for different business capabilities.
+   - Boundary of IT Solutions and IT Products are to be application capabilities so that they can enable business capabilities.
 2. IT Products are risk assessed for lock-in situations with the IT Products risk assessment framework (to be created - see [Coherent set of actions](#coherent-set-of-actions))
 
 ## Coherent set of actions
```

Comment:
On Governance / Endorse the ESDC OSS Framework, it says "Officially REVIEW the ESDC OSS Framework and endorse it."

is further review necessary? Should the strategy's action only state "seek endorsement by EARB"?

Reply:
Could adjust to that yes